### DISCUSSION TITLE
Country Exclusion in Shipping Zone

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/commerce#2296] [Ideas]
> Created by: idol chan ([idolchan@gmyle.com](idolchan@gmyle.com)) on 2016/10/17 05:01:53 +0000
Votes at time of UserVoice import: 2

------
Adding country exclusion in shipping rules/zones ? 
Or give us "elsewhere" (not "anywhere")
Say i have a US domestic shipping method that only for US, and international shipping method that for anywhere except US. Turns out US customer can choose both methods (because only shipping-zone: anywhere is available).

**Comment by lukeholder** [comment]
Moving this off to milestone 3.0, since we will probably just expand the condition formula concept to encompass more than just zip codes.
